packages/loopring_v3/contracts/aux/access/LoopringIOExchangeOwner.sol

```diff
@@ -44,6 +47,46 @@ contract LoopringIOExchangeOwner is SelectorBasedAccessManager
         }
     }
 
+    function submitBlocksWithCallbacks(
+        ExchangeData.Block[]    memory blocks,
+        ISubmitBlocksCallback[] memory callbacks,
+        uint[]                  memory blockIndices,
+        uint[]                  memory txIndices,
+        bytes[]                 memory auxiliaryData
```

Comment:
> For multiple receivers, the order to notify them doesn't matter. But for the same receiver, a strict ascent order of block indices and for each block a strict ascent order of transaction indices matters, and can be checked by a pure function that takes ReceiverNotification as input.

Not saying we need to make this completely generalized because we can always update this logic when needed, but I think this does depend on what we currently do and is not necessarily always the case (no strict 1-to-1 mapping between contract and account). Though even in the current use I think it only matters in very select cases where we need to make sure block transactions can only be used once.

So this is for optimizing the number of calls needed to a single pool right? If we think this scenario will happen a lot (which I assumed it wouldn't) this makes sense to save a bit of gas in special circumstances.